app/src/main/java/pm/gnosis/heimdall/ui/safe/overview/SafesOverviewActivity.kt

```diff
@@ -79,6 +91,16 @@ class SafesOverviewActivity : BaseActivity() {
         disposables += layout_safe_overview_add_safe.clicks()
                 .subscribeBy(onNext = { startActivity(AddSafeActivity.createIntent(this)) },
                         onError = Timber::e)
+
+        disposables += viewModel.shouldShowLowBalanceView()
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribeBy(onSuccess = ::shouldShowLowBalanceView, onError = Timber::e)
+    }
+
+    private fun shouldShowLowBalanceView(show: Boolean) {
+        (layout_safe_overview_low_balance as ExpandableLinearLayout).run {
```

Comment:
why don't you assign an id to the container of the low balance layout and access via it? you don't need to cast then.